postgres/init/21-portal-basedata.sql

```diff
@@ -516,6 +561,61 @@ COPY portal.notification_type (id, label) FROM stdin;
 \.
 
 
+--
+-- Data for Name: service_licenses; Type: TABLE DATA; Schema: portal; Owner: portal
+--
+
+COPY portal.service_licenses (id, license_text) FROM stdin;
+\.
+
+
+--
+-- Data for Name: service_statuses; Type: TABLE DATA; Schema: portal; Owner: portal
+--
+
+COPY portal.service_statuses (id, label) FROM stdin;
+1	CREATED
+2	IN_REVIEW
+3	ACTIVE
+4	INACTIVE
+\.
+
+
+--
+-- Data for Name: services; Type: TABLE DATA; Schema: portal; Owner: portal
+--
+
+COPY portal.services (id, date_created, name, thumbnail_url, provider_company_id, service_status_id, contact_email, sales_manager_id, last_editor_id) FROM stdin;
```

Comment:
@Phil91 is there a specific reason why we added services into the 21 basedata file? Ideally it would be file 22, wouldn't it?

Reply:
you're absolutly right. i moved it.